nerfstudio/scripts/exporter.py

```diff
@@ -482,6 +483,26 @@ class ExportGaussianSplat(Exporter):
     Export 3D Gaussian Splatting model to a .ply
     """
 
+    def write_ply(self, filename, count, map_to_tensors: OrderedDict):
```

Comment:
Could you add a short doc-string for this helper method? I could also see this as a class / static method since it doesn't look like relies on self.